### DISCUSSION TITLE
Obtaining ground truth data for my robot

### DISCUSSION THREAD

**Discussion by animesh-singhal** [cyberbotics/webots_ros2#744] [Q&A] [answered]
How can I accurately determine the real-world position of my robot in webots simulator? I want to compare it with its estimated position based on odometry.

**Comment by omichel** [comment, marked as answer]
Yes can add a [GPS](https://cyberbotics.com/doc/reference/gps) device on your robot with no noise (default setting) to get ground truth data on the real-world position of your robot.

**Reply by animesh-singhal** [reply]
Thanks Olivier, this helps!
